src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -159,13 +152,18 @@ public void RemovePeer(NetworkPeer peer)
             this.ConnectedPeers.Remove(peer);
         }
 
+        internal bool IsPeerConnected(IPEndPoint endPoint)
+        {
```

Comment:
I thought we are going to get rid of RelatedPeerConnector and pass the ConnectedNodes instead no?

Reply:
Yes the plan was to do this in the next PR 👍